apps/agent-tars/src/main/utils/updateApp.ts

```diff
@@ -8,27 +8,38 @@ import {
 
 export class AppUpdater {
   autoUpdater: ElectronAppUpdater = autoUpdater;
+
+  checkReleaseName(releaseName: string | undefined | null): boolean {
```

Comment:
This seems to be the essence of one repository and two applications, but you want to check updates separately